cpu/nrf5x_common/include/periph_cpu_common.h

```diff
@@ -116,6 +126,7 @@ typedef enum {
     ADC_RES_14BIT = 0xf2,   /**< ADC resolution: 14 bit */
     ADC_RES_16BIT = 0xf3    /**< ADC resolution: 16 bit */
 } adc_res_t;
+#endif
```

Comment:
looking at this now, I think we should move the typedefs into `nrf51/periph_cpu.h` and `nrf52/periph_cpu.h`. That way there is no need for an `#ifdef` anymore...